script/tutorial_1_roadmap.py

```diff
@@ -0,0 +1,80 @@
+from hpp.corbaserver.pr2 import Robot
+from hpp.corbaserver import ProblemSolver
+from hpp.gepetto import Viewer
+from hpp.gepetto import PathPlayer
+
+# define colors for the roadmap (normalized RGBA)
+white=[1.0,1.0,1.0,1.0]
+lightGreen=[0.23,0.75,0.2,0.5]
+green=[0.23,0.75,0.2,1]
+yellow=[0.85,0.75,0.15,0.7]
+blue = [0.0, 0.0, 0.8, 1.0]
+lightBlue = [0.0, 0.0, 0.8, 0.7]
+grey = [0.7,0.7,0.7,1.0]
+red = [0.8,0.0,0.0,1.0]
```

Comment:
Thanks for this tutorial !

As this is the entry point to HPP, I would propose a few changes:
- the color definitions could go in class Viewer of hpp.gepetto. It may be even better to create a IDL struct in the package `gepetto-viewer-corba`